catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -557,6 +586,33 @@ private SourceResponseImpl processResponse(InputStream is, QueryRequest queryReq
     return results;
   }
 
+  //  /**
```

Comment:
❓  Why is this chunk commented out?

Reply:
oops, sorry about that, leftovers from figuring out the rebasing